integration-tests/devtools/src/test/resources/__snapshots__/WebSocketsCodestartTest/testContent/src_main_java_ilove_quark_us_StartWebSocket.java

```diff
@@ -36,4 +36,7 @@ public void onError(Session session, @PathParam("name") String name, Throwable t
     public void onMessage(String message, @PathParam("name") String name) {
         System.out.println("onMessage> " + name + ": " + message);
     }
+
```

Comment:
I would have added a single line comment saying that the method is not necessary, BUT.... it simplify IDE integration.

Reply:
I tried different variations but couldn't find one that was not too long and didn't raise more questions. 

Got a concrete comment suggestion that you think could work?

Reply:
Something like: 'Just to run the application from IDEs'